workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/SpatialOSComponent.cs

```diff
@@ -5,20 +5,52 @@
 
 namespace Improbable.Gdk.GameObjectRepresentation
 {
+    /// <summary>
+    ///     Denotes that a GameObject has been linked to an SpatialOS entity.
```

Comment:
Does not only denote, but also allows spatialos related functionality within monobehaviour workflow